### ISSUE TITLE
python-modernize --future-unicode chokes on a complex future import statement

### ISSUE THREAD

**Issue by immerrr** [python-modernize/python-modernize#134]
I have a standard future import statement macro in my editor that involves all 2.6+ future parameters.  That macro seems to cause problems for modernize. Here's a minumim test file that fails:
```
from __future__ import (absolute_import, division,
                        print_function, unicode_literals)

hello = 'world'
```

`python-modernize --future-unicode` fails with the following error:

```
$ python-modernize ~/test.py --future-unicode
RefactoringTool: Skipping implicit fixer: idioms
RefactoringTool: Skipping implicit fixer: set_literal
RefactoringTool: Skipping implicit fixer: ws_comma
Traceback (most recent call last):
  File "/home/immerrr/.conda/envs/modernize/bin/python-modernize", line 11, in <module>
    sys.exit(main())
  File "/home/immerrr/.conda/envs/modernize/lib/python2.7/site-packages/libmodernize/main.py", line 138, in main
    options.processes)
  File "/home/immerrr/.conda/envs/modernize/lib/python2.7/lib2to3/refactor.py", line 706, in refactor
    items, write, doctests_only)
  File "/home/immerrr/.conda/envs/modernize/lib/python2.7/lib2to3/refactor.py", line 301, in refactor
    self.refactor_file(dir_or_file, write, doctests_only)
  File "/home/immerrr/.conda/envs/modernize/lib/python2.7/lib2to3/refactor.py", line 747, in refactor_file
    *args, **kwargs)
  File "/home/immerrr/.conda/envs/modernize/lib/python2.7/lib2to3/refactor.py", line 354, in refactor_file
    tree = self.refactor_string(input, filename)
  File "/home/immerrr/.conda/envs/modernize/lib/python2.7/lib2to3/refactor.py", line 386, in refactor_string
    self.refactor_tree(tree, name)
  File "/home/immerrr/.conda/envs/modernize/lib/python2.7/lib2to3/refactor.py", line 460, in refactor_tree
    new = fixer.transform(node, results)
  File "/home/immerrr/.conda/envs/modernize/lib/python2.7/site-packages/libmodernize/fixes/fix_unicode_future.py", line 10, in transform
    add_future(node, 'unicode_literals')
  File "/home/immerrr/.conda/envs/modernize/lib/python2.7/site-packages/libmodernize/__init__.py", line 53, in add_future
    names = check_future_import(node)
  File "/home/immerrr/.conda/envs/modernize/lib/python2.7/site-packages/libmodernize/__init__.py", line 42, in check_future_import
    assert 0, "strange import"
AssertionError: strange import
$ python-modernize --version
modernize 0.4
```

**Comment by takluyver** [comment]
Can you try with master? I think this may have been fixed by #117.

**Comment by immerrr** [comment]
Master seems to work, thank you.
